config/serverless.oblt.yml

```diff
@@ -27,6 +27,14 @@ xpack.apm.serverlessOnboarding: true
 xpack.fleet.packages:
   - name: apm
     version: latest
+## Disable APM UI components and API calls
+xpack.apm.featureFlags.agentConfigurationAvailable: false
+xpack.apm.featureFlags.configurableIndicesAvailable: false
+xpack.apm.featureFlags.infrastructureTabAvailable: false
+xpack.apm.featureFlags.infraUiAvailable: false
+xpack.apm.featureFlags.migrationToFleetAvailable: false
+xpack.apm.featureFlags.sourcemapApiAvailable: false
+xpack.apm.featureFlags.storageExplorerAvailable: false
```

Comment:
NIT: we could use the YAML tree format to avoid repetition 😇 
```suggestion
## Disable APM UI components and API calls
xpack.apm.featureFlags:
	agentConfigurationAvailable: false
	configurableIndicesAvailable: false
	infrastructureTabAvailable: false
	infraUiAvailable: false
	migrationToFleetAvailable: false
	sourcemapApiAvailable: false
	storageExplorerAvailable: false
```